### DISCUSSION TITLE
Use Tailwind

### DISCUSSION THREAD

**Discussion by dsousa12** [facebook/docusaurus#7830] [Q&A]
Hello everyone.
I'm trying to use **TailwindCSS** on my project, but I have only one problem and it's related with dark mode.

```js
module.exports = {
darkMode: '[data-theme="dark"]',
}
```

I try that code in my `tailwindcss.config.js`, but not working.
Anyone already have dark theme working with **TailwindCSS**?

**Comment by arisac** [comment]
```js
module.exports = {
  darkMode: ['class', '[data-theme="dark"]'],
}
```
